src/theory/quantifiers/sygus/sygus_pbe.cpp

```diff
@@ -205,36 +205,69 @@ bool CegConjecturePbe::initialize(Node n,
       d_enum_to_active_guard[e] = g;
       d_enum_to_candidate[e] = c;
       TNode te = e;
-      // initialize static symmetry breaking lemmas for it
+      // initialize static and dynamic symmetry breaking lemmas for it
       // we register only one "master" enumerator per type
       // thus, the strategy lemmas (which are for individual strategy points)
       // are applicable (disjunctively) to the master enumerator
       std::map<TypeNode, std::vector<Node>>::iterator itt =
           tn_to_strategy_pt.find(etn);
       if (itt != tn_to_strategy_pt.end())
       {
-        std::vector<Node> disj;
+        std::vector<Node> disj_static;
+        std::map<TypeNode, std::pair<std::vector<Node>, unsigned>> disj_dynamic;
         for (const Node& sp : itt->second)
         {
-          std::map<Node, std::vector<Node>>::iterator itsl =
+          std::map<Node, StrategyRedundancies>::iterator itsl =
               strategy_lemmas.find(sp);
           Assert(itsl != strategy_lemmas.end());
-          if (!itsl->second.empty())
+          if (!itsl->second.d_simple_lemmas.empty())
           {
             TNode tsp = sp;
-            Node lem = itsl->second.size() == 1 ? itsl->second[0]
-                                                : nm->mkNode(AND, itsl->second);
+            Node lem = itsl->second.d_simple_lemmas.size() == 1
+                           ? itsl->second.d_simple_lemmas[0]
+                           : nm->mkNode(AND, itsl->second.d_simple_lemmas);
             if (tsp != te)
             {
               lem = lem.substitute(tsp, te);
             }
-            disj.push_back(lem);
+            disj_static.push_back(lem);
```

Comment:
I think we're actually missing an else here (and below) in the case that there are no simple/dynamic lemmas.